airflow/providers/google/cloud/operators/pubsub.py

```diff
@@ -196,8 +196,7 @@ class PubSubCreateSubscriptionOperator(BaseOperator):
 
         with DAG('failing DAG') as dag:
             (
-                dag
-                >> PubSubSubscriptionCreateOperator(
+                PubSubSubscriptionCreateOperator(
```

Comment:
FYI, still have this syntax in line 44, 56, 185, 373, 382, 485, 496

Reply:
Fixed